home/templates/home/_featured_incidents_section.html

```diff
@@ -1,21 +1,15 @@
 {% load wagtailcore_tags %}
 
-<h2 class="homepage-section-header">Featured Incidents</h2>
+<h2 class="homepage-section-header">{{ label }}</h2>
```

Comment:
In this template and similar template files, can we have a default value if label is not mentioned? As is {{ label|default:"Featured Incidents" }} ?

Reply:
Good idea. I have added them.